### ISSUE TITLE
No padding in rpaths?

### ISSUE THREAD

**Issue by haampie** [alalazo/spack-bootstrap-mirrors#10]
Seems like there is no padding in rpaths in the binaries here, which complicates relocation

**Comment by alalazo** [comment]
This should be solved in the latest tarballs. Previously only GnuPG was compiled with padding.

**Comment by alalazo** [comment]
Can you define "complicates"? I am wondering if there's any issue that I missed to double check for the new binary tarball.

**Comment by haampie** [comment]
It requires all the complicated machinery of adding a new section to an elf file, which is very buggy in many standard distro versions of patchelf.

It's better to do in-place patching of rpaths, since that's trivial and unlikely to result in broken binaries.

**Comment by alalazo** [comment]
Ok, thanks for clarifying. So this is in preparation to have binaries used with a feature that is not yet in.

**Comment by haampie** [comment]
Feel free to close 🙃

**Comment by alalazo** [comment]
Sure, I think I'll wait until this is deployed and then  close.  Thanks.

**Comment by haampie** [comment]
This was done right?

**Comment by alalazo** [comment]
Yes, but not deployed yet. In any case if I recall correctly this was missing only in `clingo` (not in `GnuPG`) and `clingo` doesn't have `prefix` in `bss` or similar places. If you want to test drive the latest binaries they are [here](https://github.com/alalazo/spack-bootstrap-mirrors/releases/tag/v0.2-rc2)
